.github/workflows/security-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Security tests
+# This workflow is triggered on pull requests to main branch
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    # Job name
+    name: Build Replication plugin
+    runs-on: ubuntu-latest
+    steps:
+      # This step uses the setup-java Github action: https://github.com/actions/setup-java
+      - name: Set Up JDK 14
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+      # This step uses the checkout Github action: https://github.com/actions/checkout
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+      # Security plugin dependency
+      - name: Checkout security
+        uses: actions/checkout@v2
+        with:
+          repository: 'opensearch-project/security'
+          path: security
+          ref: 'main'
+      - name: Build security
+        working-directory: ./security
+        run: |
+          mvn clean package -Padvanced -DskipTests
+          cp target/releases/opensearch-security-*-SNAPSHOT.zip ../src/test/resources/security/plugin/
```

Comment:
Do we have an issue open for this? I understand why you are doing it, but you should be using an artifact produced by CI instead of rebuilding security so that someone coming to this project can just check out the code and build it without any kind of prerequisites.

Reply:
we are tracking the move to maven distributed version of plugin builds for replication under this issue: https://github.com/opensearch-project/cross-cluster-replication/issues/274 which is currently dependent o on https://github.com/opensearch-project/opensearch-build/issues/716

For the local, security plugin build is not mandatory - Developers can still independently checkout replication repo, build and run locally.